### PR TITLE
blog and figure

### PR DESCRIPTION
**Pull request by thigm85**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: _posts/2021-03-29-build-news-recommendation-app-from-python-with-vespa.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+## Feeding and partial updates: news and user embeddings
+
+To keep this tutorial easy to follow, we make the parsed embeddings available for download. To build them yourself, please follow [this tutorial](https://docs.vespa.ai/en/tutorials/news-4-embeddings.html).
+
+
+```python
+import requests, json
+
+user_embeddings = json.loads(
+    requests.get("https://thigm85.github.io/data/mind/mind_demo_user_embeddings_parsed.json").text
```
Comment (kkraune) [inline _posts/2021-03-29-build-news-recommendation-app-from-python-with-vespa.md:173]: should we put these files somewhere in https://github.com/vespa-engine/sample-apps/tree/master/news instead?

**Review by kkraune — APPROVED** [review]
consider comment on file location, otherwise great post!

File: _posts/2021-03-29-build-news-recommendation-app-from-python-with-vespa.md (hunk 1 above)
Comment (kkraune) [reply _posts/2021-03-29-build-news-recommendation-app-from-python-with-vespa.md:173]: agreed. we don't want to bload the size of the sampleapp repo. lets keep this for now, can always change later

Comment (thigm85) [reply _posts/2021-03-29-build-news-recommendation-app-from-python-with-vespa.md:173]: @freva how can I use it for future posts? Any pointers?

**Pull request merged**